sonaranalyzer-dotnet/rspec/cs/S4564_c#.json

```diff
@@ -7,20 +7,10 @@
     "constantCost": "5min"
   },
   "tags": [
-    "cwe",
```

Comment:
this rule https://jira.sonarsource.com/browse/RSPEC-4564 got deprecated, hence the removal of standards